README.md

```diff
@@ -203,6 +204,7 @@ Asynchronous concurrent distributed networking framework for PHP
 ### HTTP
 
 * [amphp/artax](https://github.com/amphp/artax) - An async HTTP/1.1 client built on [amp](#amphp)
+* [amphp/aerys](https://github.com/amphp/aerys) - A non-blocking HTTP application, WebSocket and file server
```

Comment:
Put it here because the react server is also listed here, not only clients. I'm not sure why it's listed as web application after all. What should be listed there? Just applications? Then we have to move the other WebSocket libraries as well.